db_upgrade_scripts/mosip_master/ddl/master-dynamic_field.sql

```diff
@@ -26,8 +26,7 @@ CREATE TABLE master.dynamic_field(
 	upd_dtimes timestamp,
 	is_deleted boolean,
 	del_dtimes timestamp,
-	CONSTRAINT pk_schfld_id PRIMARY KEY (id),
-	CONSTRAINT uk_schfld_name UNIQUE (name,lang_code)
```

Comment:
we don't need this unique constraint ?

Reply:
Removed this DDL, we have moved this DDL with valid constraints to dynamic field migration script.